soc/openhwgroup/cva6/Kconfig.defconfig

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2024 CISPA Helmholtz Center for Information Security gGmbH
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_FAMILY_OPENHWGROUP_CVA6
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 25000000
+
+config RISCV_SOC_INTERRUPT_INIT
+	default y
+
+# highest implemeneted first-level interrupt is machine external, 11
```

Comment:
Capital first letters for comments, also typo `# Highest implemented first-level interrupt is machine external, 11` fix throughout whole PR